zebrad/tests/acceptance.rs

```diff
@@ -1212,6 +1212,77 @@ async fn rpc_endpoint() -> Result<()> {
     Ok(())
 }
 
+#[test]
+fn non_blocking_logger() -> Result<()> {
+    use futures::FutureExt as _;
+
+    let rt = tokio::runtime::Runtime::new().unwrap();
+
+    let test_task_handle: tokio::task::JoinHandle<Result<()>> = rt.spawn(async {
+        let _init_guard = zebra_test::init();
+
+        // Write a configuration that has RPC listen_addr set
+        // [Note on port conflict](#Note on port conflict)
+        let mut config = random_known_rpc_port_config()?;
+        config.tracing.filter = Some("trace".to_string());
+        config.tracing.buffer_limit = 100;
+        let zebra_rpc_address = config.rpc.listen_addr.unwrap();
+
+        let dir = testdir()?.with_config(&mut config)?;
+        let mut child = dir.spawn_child(args!["start"])?;
+        // Wait until port is open.
+        child.expect_stdout_line_matches(
+            format!("Opened RPC endpoint at {}", config.rpc.listen_addr.unwrap()).as_str(),
+        )?;
+
+        let mut cmd2 = std::process::Command::new("echo")
```

Comment:
I'm not sure if this extra task is needed for the test. So once we have the tests passing, we might want to delete it.

Reply:
It was not needed, deleted in `[590bb91](https://github.com/ZcashFoundation/zebra/pull/5032/commits/590bb914f65ffd8fd0da57446077bf6f6f644e21)`